hw07/6.py

```diff
@@ -0,0 +1,15 @@
+# Создать lambda функцию, которая принимает на вход
+# неопределенное количество именных аргументов и
+# выводит словарь с ключами удвоенно длины. {‘abc’:
+# 5} -> {‘abcabc’: 5}
+
+
+def main():
+    dict = ['dsfs': 3, 'fdg2r': 34, 'erqwe': 432]
```

Comment:
Это что? Если словарь, то он определяется не через квадратные скобки.

Зачем тут вообще map? Должно быть как-то так:

```suggestion
    my_func = lambda<тут_все_что_она_должна_принимать_и_делать>
    print(my_func(maks='lopuh', chitat='vnimatelno'))
```